### ISSUE TITLE
Angular degree search - pagination buttons have mismatched, redundant `aria-label`s

### ISSUE THREAD

**Issue by cjg89** [UCF/UCF-Degree-Search-Plugin#108]
**Describe the bug**

Pagination for the Angular degree search has redundant and mismatched aria-labels:

https://github.com/UCF/UCF-Degree-Search-Plugin/blob/abfcc55c27ddd1a2cdf6075f2ec6ec74ff2504b4/includes/ucf-degree-search-angular-common.php#L245-L272

Particularly, the "next" button has an `aria-label` of "Previous".

**Steps to Reproduce**

n/a

**Expected Behavior**

These pagination buttons should have a single text label that is accurate to the intended function.

**Possible Fix**

Remove `aria-label` attributes; they are unnecessary since we're already using inner `.sr-only` spans.

**Your environment:**

n/a

**Additional context**

n/a
